weave-js/src/components/Panel2/PanelFilterEditor.tsx

```diff
@@ -649,6 +678,18 @@ export const PanelFilterEditor: React.FC<PanelFilterEditorProps> = props => {
     [listItem.type, updateVal]
   );
 
+  const pushTempVisualClauses = useCallback(() => {
```

Comment:
push makes me think of a queue or a stack, though I see why you called it that. How about "commit"?

Reply:
make sense